code/modules/research/xenobiology/xenobio_camera.dm

```diff
@@ -87,6 +140,13 @@
 		if(loaded)
 			to_chat(user, "<span class='notice'>You fill [src] with the monkey cubes stored in [O]. [src] now has [monkeys] monkey cubes stored.</span>")
 		return
+	else if (istype(O, /obj/item/multitool))
+		var/obj/item/multitool/I = O
+		if (istype(I.buffer, /obj/machinery/monkey_recycler))
+			to_chat(user, "<span class='notice'>You link [src] with [I.buffer] in [I] buffer.</span>")
+			connected_recycler = I.buffer
+			connected_recycler.connected += src
+			return
 	..()
```

Comment:
this need to end with
```
else
	return ..()
```

Reply:
Does that still apply when I do in the multitool_act proc?

Reply:
We don't have a `multitool_act` proc.

Reply:
so what's "the helper"?